nautobot/extras/templates/extras/marketplace.html

```diff
@@ -0,0 +1,227 @@
+{% extends "base.html" %}
+{% load helpers %}
+{% load static %}
+
+{% block extra_styles %}
+<style>
+    a.tile,
+    a.tile:hover,
+    a.tile:focus,
+    a.tile:visited {
+        color: inherit;
+        text-decoration: none;
+    }
+
+    .tile-header {
+        display: block;
+    }
```

Comment:
Is there a reason these styles aren't in `base.css`?

Reply:
all styles here are page specific, I prefer to keep them scoped to the page not to bloat global context with styles unused anywhere else